README.md

```diff
@@ -39,41 +39,63 @@ we would like things to flow.
 ### Requirements
 
 * [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
-* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* OpenJDK 8
 * Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
 
 Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-BundleDevelopmentOverview)
 
-NOTE: Version 0.0.2+ of Conan is only compatible with Nexus Repo 3.11.0 or greater due to adding Tree View Filtering. If 
-you need to use this plugin with an older version, please use 0.0.1
-
 ### Building
 
 To build the project and generate the bundle use Maven
 
-    mvn clean package -PbuildKar
+    mvn clean install
 
 If everything checks out, the bundle for Conan should be available in the `target` folder
 
-#### Build with Docker
+## Using Conan With Nexus Repository Manager 3
 
-`docker build -t nexus-repository-conan:0.0.6 .`
+[We have detailed instructions on how to get started here!](https://help.sonatype.com/nxrm3master/formats/conan-repositories)
 
-#### Run as a Docker container
+## Compatibility with Nexus Repository Manager 3 Versions
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
+The table below outlines what version of Nexus Repository Manager the plugin was built against
 
-For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+| Plugin Version    | Nexus Repository Manager Version |
+|-------------------|----------------------------------|
+| v0.0.1            | <3.11.0                          |
+| v0.0.2+           | 3.11.0+                          |
+| v1.0.0+ In product| 3.22.0+                          |
 
-The application will now be available from your browser at http://localhost:8081
 
-## Using Conan With Nexus Repository Manager 3
+## Installing the plugin
 
-[We have detailed instructions on how to get started here!](docs/CONAN_USER_DOCUMENTATION.md)
+In Nexus Repository Manager 3.22+ Conan proxy format is already included. So there is no need to install it.
```

Comment:
NOTE: "Conan proxy format" sounds weird to me, because proxy isn't a format.  It's like you're trying to say 2 things here...Conan format and proxy.  It's hard to squeeze into 1 phrase, maybe "Conan format with proxy".  Easier might just be saying "Conan proxy" (no format).

Trivial: I think you can just make this one sentence "is already included, so".

Reply:
Fixed e3ab48c08203df139ed0311ab8e2868eaa9f1b76